lib/go-atscfg/headerrewritedotconfig.go

```diff
@@ -35,7 +35,7 @@ const HeaderRewritePrefix = "hdr_rw_"
 const ContentTypeHeaderRewriteDotConfig = ContentTypeTextASCII
 const LineCommentHeaderRewriteDotConfig = LineCommentHash
 
-const ServiceCategoryHeader = "CDN-SVC"
+const ServiceCategoryHeader = "X-CDN-SVC"
```

Comment:
Why `X-`? 

The IETF has asked people to stop prefixing headers with `X-` because it makes it hard to remove in a standard later when they become common: https://tools.ietf.org/html/rfc6648